tortoise/util.go

```diff
@@ -81,26 +75,28 @@ func (a vec) String() string {
 	return "abstain"
 }
 
-func calculateOpinionWithThreshold(logger log.Log, v vec, layerSize int, theta uint8, delta float64) vec {
-	threshold := float64(theta) / 100 * delta * float64(layerSize)
-	netVote := float64(v.netVote())
+func calculateOpinionWithThreshold(logger log.Log, v vec, layerSize int, theta uint8, delta uint32) vec {
+	threshold := big.NewInt(int64(theta))
+	threshold.Mul(threshold, big.NewInt(int64(delta)))
```

Comment:
delta and layerSize are uint32 (thought layerSize is casted to int in the upper). i am casting to int64 because big.NewInt accepts int64 in the initializer, and since the value always fits in 63 bits this is not a problem. SetUint64 is also correct but there is literally not difference